specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,875 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SignalR Service REST API",
+    "version": "v1"
+  },
+  "paths": {
+    "/api/v1/health": {
+      "head": {
+        "tags": [
+          "general"
+        ],
+        "summary": "Get service health status.",
+        "operationId": "HealthApiIndex_Head",
```

Comment:
Operation name should not show the implementation detail of the HTTP layer. We could call it `HealthApiIndex_GetServiceHealthStatus` for instance to be more explicit. The API review board might have stronger naming opinion.